fixed-int.md

```diff
@@ -0,0 +1,44 @@
+```k
+module FIXED-INT
+    imports INT
+    imports BOOL
+
+    syntax FInt ::= "(" FInt ")"                   [bracket]
+                  | FInt ( value: Int , one: Int ) [klabel(FInt), symbol]
+ // ---------------------------------------------------------------------
+    // rule #Ceil(FInt(... value: V, one: O)) => { V >=Int 0 andBool O >Int 0 #Equals true } [anywhere, symbolic]
```

Comment:
So this is commented out...will it eventually be uncommented, or can we get rid of it?

If we're keeping it:
1) is the "`#Equals true`" part really necessary? Looks like `V >=Int 0 andBool O >Int 0` should already evaluate to a boolean.
2) as I was once told on a topology homework assignment, the letter O is an unfortunate choice for a variable name...

Reply:
Well, we _should_ have this definedness axiom present, but right now the LLVM backend is rejecting it. We have to wait for some changes to the way functions work in K though before it can stop rejecting it.

So I'll remove the axiom, and I've renamed all the variables `O* => ONE*` and `V* => VALUE*`.